src/reducers/related-views.ts

```diff
@@ -4,11 +4,11 @@ import { DEFAULT_RELATED_VIEWS, RelatedViews } from '../models/related-views';
 
 export function relatedViewsReducer(
   relatedViewToggler: RelatedViews = DEFAULT_RELATED_VIEWS, action: Action): RelatedViews {
-  const { isHidden } = relatedViewToggler;
   switch (action.type) {
     case RELATED_VIEWS_HIDE_TOGGLE: {
+      const {hideRelatedViews} = action.payload;
```

Comment:
`hideRelatedViews` here is "current" status of hideRelatedViews that you want to switch form.  The current name doesn't make it clear whether it is an old value (that you don't want) or a new value that you want.

I'd say since this is an action, better make rename `hideRelatedViews` and make it `newIsHidden` state (not `currentIsHidden`).